pkgs/desktops/gnome/devtools/gnome-devel-docs/default.nix

```diff
@@ -17,10 +17,10 @@ stdenv.mkDerivation rec {
   buildInputs = [ itstool libxml2 ];
 
   meta = with lib; {
-    homepage = "https://github.com/GNOME/gnome-devel-docs";
+    homepage = "https://gitlab.gnome.org/Archive/gnome-devel-docs";
     description = "Developer documentation for GNOME";
     maintainers = teams.gnome.members;
-    license = licenses.fdl12;
+    license = licenses.fdl11Plus;
```

Comment:
Looks good, though some parts of the package [use CC instead](https://gitlab.gnome.org/Archive/gnome-devel-docs/-/blob/master/COPYING). But I would not bother, we should remove this package anyway, as it is no longer relevant to the GNOME platform.

Reply:
Rebased.